test/test_PostRequestHandler.cpp

```diff
@@ -0,0 +1,80 @@
+#include "gmock/gmock.h"
+#include <gtest/gtest.h>
+
+#include <stdexcept>
+#include <string>
+
+#include "MockFileSystemPolicy.hpp"
+
+#include "PostRequestHandler.hpp"
+
+using ::testing::NiceMock;
+using ::testing::Return;
+
+class PostRequestHandlerTest : public ::testing::Test {
+protected:
+	PostRequestHandlerTest() {
+
+		fakeStat.st_size = 1024;
+		fakeStat.st_mtime = 777;
+
+		ON_CALL(fileSystemPolicy, writeToFile)
+		.WillByDefault(testing::Return());
+		ON_CALL(fileSystemPolicy, getFileStat)
+		.WillByDefault(testing::Return(fakeStat));
+	 }
+	~PostRequestHandlerTest() override { }
+
+	std::string path = "/workspaces/webserv/test/";
+	std::string content = "Hello, World!";
+	struct stat fakeStat = {};
+    
+	NiceMock<MockFileSystemPolicy> fileSystemPolicy;
```

Comment:
I started doing also m_ for the members of the test class. Has the benefit of seeing what variable is provided by test class/fixture and what is only specific to one test

Reply:
Sure, why not, changed